docs/src/main/tut/docs/streams.md

```diff
@@ -5,10 +5,12 @@ title: "Streams"
 
 # Streams
 
-Starting version `0.5.0`, you can stream the query result. As of `0.5.0`, there is only one implementation of streaming - Akka Stream.
+Starting version `0.5.0`, you can stream the query result. As of `?.?.?`, there are two implementations of streaming - [Akka Streams](https://doc.akka.io/docs/akka/current/stream/index.html) & [FS2](https://fs2.io/).
```

Comment:
Remember to change the version number when releasing 😉 

Also, since I do not use Akka Streams, I am not sure if that is the best link.